package.json

```diff
@@ -8,7 +8,7 @@
     "email": "containers@suse.com"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=18.16"
```

Comment:
Please update `README.md` to say this node version. (There are multiple places within that file.)